### ISSUE TITLE
Support for vim8 async

### ISSUE THREAD

**Issue by kiryph** [mrtazz/simplenote.vim#88]
vim8 will be released very soon (http://www.vim.org/). So it also would be great to have async support for vim8 not only for neovim.

**Comment by atomicules** [comment]
Thanks for the heads up.

**Comment by atomicules** [comment]
Waiting on vim8 for NetBSD/Pkgsrc.

**Comment by atomicules** [comment]
Closing as a duplicate of #66 because that was here first.
